packages/react-badge/src/components/Badge/Badge.types.ts

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import { ComponentProps, ShorthandProps, ObjectShorthandProps } from '@fluentui/react-utils';
+import { SizeValue } from '@fluentui/react-theme';
+
+/**
+ * {@docCategory Badge\}
+ */
+export type BadgeStatus =
+  | 'success'
+  | 'warning'
+  | 'severe'
+  | 'accent'
+  | 'danger'
+  | 'important'
+  | 'informative'
+  | 'subtle';
+
+/**
+ * {@docCategory Badge\}
+ */
+export type BadgeVariant = 'filled' | 'outline' | 'ghost' | 'inverted' | 'tint';
+
+/**
+ * {@docCategory Badge\}
+ */
+export type BadgeShape = 'rounded' | 'square' | 'circular';
+
+/**
+ * {@docCategory Badge\}
+ */
+export interface BadgeProps extends ComponentProps, React.HTMLAttributes<HTMLElement> {
+  /**
+   * A Badge can be sized.
+   */
+  size?: SizeValue;
+
+  /**
+   * A Badge can have status styles variants
+   */
+  status?: BadgeStatus;
+
+  /**
+   * A Badge can have different colors.
+   */
+  color?: string;
```

Comment:
If we support a `color` prop, we'd also need a `backgroundColor` and `borderColor` prop. That said, doing that would make it not possible to do the appearance variants like `ghost` or `tint`.

Reply:
My bad, removed